apps/address-service/domains/address/schema/Address.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Generated by `createschema address.Address 'source:Text; address:Text; key:Text; meta:Json'`
+ */
+
+const { Text, Relationship } = require('@keystonejs/fields')
+const { Json } = require('@open-condo/keystone/fields')
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+const { historical, versioned, uuided, tracked, softDeleted } = require('@open-condo/keystone/plugins')
+const access = require('@address-service/domains/address/access/Address')
+const { dvAndSender } = require('@open-condo/keystone/plugins/dvAndSender')
+
+const Address = new GQLListSchema('Address', {
+    schemaDoc: 'A model containing data on the particular building\'s address',
+    labelResolver: ({ address }) => address,
+    fields: {
+
+        sources: {
+            schemaDoc: 'The origin which used to detect the address',
+            type: Relationship,
+            ref: 'AddressSource.address',
+            many: true,
```

Comment:
! allAddress {
   source{ name id}
 }

Query complexity: `O(n * m)`

Reply:
it's better to use AddressSource (where: {...})

Reply:
Refactored